tests/test_chem.f90

```diff
@@ -0,0 +1,14 @@
+program test_chem
+   use CATChem, only: cc_get_micm_version
+   use testing_mod, only: assert
+   implicit none
+
+   character(len=:), allocatable :: micm_version
+   character(len=*), parameter :: expected_micm_version = "3.7.0"
+
+   micm_version = adjustl(trim(cc_get_micm_version()))
```

Comment:
The `adjustl` isn't really needed, but to work properly it should be in this order:
```suggestion
   micm_version = trim(adjustl(cc_get_micm_version()))
```